guides/micronaut-azure-cloud/micronaut-azure-cloud.adoc

```diff
@@ -0,0 +1,202 @@
+= @guideTitle@
+
+@guideIntro@
+
+Authors: @authors@
+
+Micronaut Version: @micronaut@
+
+include::{commondir}/common-gettingStarted.adoc[]
+
+== What you will need
+
+To complete this guide, you will need the following:
+
+* Some time on your hands
+* A decent text editor or IDE
+* https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest?WT.mc_id=opensource-micronaut-brborges[Azure CLI] installed and https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli?WT.mc_id=opensource-micronaut-brborges&view=azure-cli-latest#sign-in-interactively[authenticated]
+* JDK 8 installed with JAVA_HOME configured appropriately (cannot be Java 9 or later)
+
+=== Costs
+
+This guide uses services available in the https://azure.microsoft.com/free/free-account-faq/?WT.mc_id=opensource-micronaut-brborges[Always Free Tier] of https://azure.microsoft.com/free/?WT.mc_id=opensource-micronaut-brborges[Microsoft Azure Free Account]. If you don't have one already, https://azure.microsoft.com/free/?WT.mc_id=opensource-micronaut-brborges[create your account] to complete this guide.
+
+
+include::{commondir}/common-completesolution.adoc[]
+
+== Writing the App
+
+Create an app using the http://docs.micronaut.io/latest/guide/index.html#cli[Micronaut Command Line Interface] or with
+https://launch.micronaut.io[Micronaut Launch].
+
+[source,bash]
+----
+mn create-app example.micronaut.micronautguide --build=@build@ --lang=@lang@
+----
+
+NOTE: If you don't specify the `--build` argument, Gradle is used as a build tool. +++<br/>+++ If you don't specify the `--lang` argument, Java is used as a language. For this guide, Maven is the required build tool.
+
+The previous command creates a micronaut app with the default package `example.micronaut` in a folder named `micronautguide`.
+
+include::{commondir}/common-annotationprocessors.adoc[]
+
+== How to Complete
+
+To get started do the following:
+
+Download and unzip the source or clone the following GitHub repository:
+
+[source, bash]
+----
+git clone https://github.com/micronaut-guides/micronaut-azure-cloud.git
+----
```

Comment:
This shouldn't be necessary. The users either download the .zip of the full project or create the app step by step.